eng/pipelines/coreclr/templates/crossgen2-comparison-job.yml

```diff
@@ -154,6 +157,7 @@ jobs:
             --diff_dir   $HELIX_WORKITEM_PAYLOAD/log
         ${{ if eq(parameters.osGroup, 'windows') }}:
           WorkItemCommand:
+            echo $(displayname_comparison_job)
```

Comment:
Not sure how this is set up, but does this line need a trailing `&` like the commands that follow? (Looks like this is set to create one long single-line batch script?)

Reply:
It should have the &, but it does work. See https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-44119-head-8c5be3842f80486d85/WorkItem/console.9e397036.log